lib/pending_assigns_reminder.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+class PendingAssignsReminder
+  REMINDED_AT = 'last_reminded_at'
+  REMINDER_THRESHOLD = 2
+
+  def remind(user)
+    newest_topics = assigned_topics(:newest, user)
+    return if newest_topics.size < REMINDER_THRESHOLD
+    oldest_topics = assigned_topics(:oldest, user).where.not(id: newest_topics.map(&:id))
```

Comment:
Personally, I would go with something like `assigned_topics(user, order: :asc).where.not(id: newest_topics.map(&:id))`

`def assigned_topics(user, order: :desc)`

I don't feel like we need `:newest, :oldest` to represent ordering here.